toolchain/check/operator.cpp

```diff
@@ -35,7 +35,8 @@ auto BuildUnaryOperator(Context& context, SemIR::LocId loc_id, Operator op,
                         MakeDiagnosticBuilderFn missing_impl_diagnoser)
     -> SemIR::InstId {
   // Look up the operator function.
-  auto op_fn = GetOperatorOpFunction(context, loc_id.ToImplicit(), op);
+  auto op_fn = GetOperatorOpFunction(
+      context, context.insts().GetResolvedLocId(loc_id).ToImplicit(), op);
```

Comment:
Here, you're changing both calls to `GetOperatorOpFunction`: maybe that could be changed to do this `ToImplicit`? (I think the `ToImplicit` could have been there before, it's just getting more verbose now)

Reply:
Done